test/integration/targets/bigip_gtm_server/tasks/main.yaml

```diff
@@ -139,3 +139,5 @@
   loop:
     - "{{ dc_name1 }}"
     - "{{ dc_name2 }}"
+
+- import_tasks: issue-00513.yaml
```

Comment:
Add a `tags` line underneath this import so that we can test these out-of-band of the entire integration suite.

for example,

`tags: issue-00513`

Reply:
Done.

Reply:
Done.